src/rules/objectLiteralSortKeysRule.ts

```diff
@@ -62,10 +62,12 @@ export class Rule extends Lint.Rules.OptionallyTypedRule {
     public static FAILURE_STRING_ALPHABETICAL(name: string): string {
         return `The key '${name}' is not sorted alphabetically`;
     }
+    /* tslint:disable:no-shadowed-variable */
```

Comment:
sorry, master was broken, please revert these lines and merge master with https://github.com/palantir/tslint/pull/3118